aws/resource_aws_config_delivery_channel.go

```diff
@@ -162,7 +162,18 @@ func resourceAwsConfigDeliveryChannelDelete(d *schema.ResourceData, meta interfa
 	input := configservice.DeleteDeliveryChannelInput{
 		DeliveryChannelName: aws.String(d.Id()),
 	}
-	_, err := conn.DeleteDeliveryChannel(&input)
+
+	err := resource.Retry(30*time.Second, func() *resource.RetryError {
+		_, err := conn.DeleteDeliveryChannel(&input)
+		if err != nil {
+			if isAWSErr(err, "LastDeliveryChannelDeleteFailedException", "there is a running configuration recorder") {
```

Comment:
Nitpick: This is available as a constant: `configservice.ErrCodeLastDeliveryChannelDeleteFailedException`

Reply:
Good point, swapped.